apostrophecms/import-export
# Pro 6387 images errors

[PRO-6387](https://linear.app/apostrophecms/issue/PRO-6387/[import-export]-images-not-displayed-after-import-in)

## Summary

* Import attachments first to avoid them being stripped (Also when overridding)
* Uses new `simulateRelationshipsFromStorage` method to simulate relationship data on database documents.
* Uses new `fetchRelationships` options on convert method to avoid removing relationships if not already available.
* When importing singletons, they are not overridden by default but we ask user if he wants to override them (works only if the ID is the same there is another ticket to make it more robust, it occurs because we now pre check for duplicates).

## What are the specific steps to test this change?

* Import documents with attachments for the first time it should work, relationships should stay.
* Import singleton with same ID, it should ask for override

Cypress tests running [HERE](https://github.com/apostrophecms/testbed/actions/runs/10420873077)  🟢 

## What kind of change does this PR introduce?


## Make sure the PR fulfills these requirements:
